scripts/SasLowNoisePolicy.psm1

```
@@ -12,29 +12,105 @@ scope, ports, rate, retries, freshness, evidence reuse, and staging only justifi
+    try {
+        $document = Get-Content -LiteralPath $path -Raw -ErrorAction Stop | ConvertFrom-Json
+    } catch {
+        throw "Canonical low-noise policy is invalid: $path; $($_.Exception.Message)"
+    }
+    if ($document.schema_version -ne 'sas-low-noise-policy/v1' -or -not $document.profiles) {
+        throw "Canonical low-noise policy has unsupported schema or no profiles: $path"
+    }
+    return $document
+}
+
 function Get-SasLowNoisePolicy {
```

PSScriptAnalyzer (1.25.0)

[info] The cmdlet 'Get-SasLowNoisePolicy' does not have a help comment.

(PSProvideCommentHelp)